packages/assets/src/buildAssetProfile.js

```diff
@@ -183,14 +183,17 @@ function buildAssetProfile(assetInfo, expandedRegistryProfile) {
       component.rootNodeName = componentId;
       component.visualResponses = buildDefaultVisualResponses(componentId, component.type);
       if (component.type === 'touchpad') {
-        component.touchPointNodeName = `${componentId}-axes-touched-value`;
+        component.touchPointNodeName = `${componentId}_axes_touched_value`;
       }
     });
   });
 
   // Override any properties enumerated in the asset description file
   applyAssetOverrides(profile, assetInfo);
 
+  // Replace all the "-" in component and asset names with the Maya compatible "_"
```

Comment:
Is this the right approach, or should the components/assets be expected to use _ and any use of - is considered an error?

Reply:
Any use of "-" is an error.   Once i made the readme changes, this approach looked ooky though.  I've made a fix and as a result I've also changed the asset override schema to only accept "_"